content/monitors/notifications.md

```diff
@@ -134,41 +134,57 @@ The @ notification inside the template variables follows the same rules.
 
 #### `{{is_match}}` or `{{is_exact_match}}`
 
-The `{{is_match}}` conditional allows you to match the triggering context to some string to display a different message.
-For example, you might want to notify your db team if a triggering host has `role:db` but notify your app team if the host has `role:app`.
+##### {{is_match}} 
 
-You can use any of the available tag variables in your condition. A match is made if the comparison string is anywhere in the resolved variable.
-
-The variable uses the following format:
+The `{{is_match}}` conditional allows you to match the triggering context to some string to display a different message. You can use any of the available tag variables in your condition. A match is made if the comparison string is anywhere in the resolved variable. The variable uses the following format:
```

Comment:
The `{{is_match}}` conditional allows you to match the triggering context to some string to display a different message. You can use any of the available tag variables in your conditional statement. A match is made if the comparison string is anywhere in the resolved variable. Variables use the following format:

Reply:
"some string" is sort of weird, no? Possibly consider:
* a string
* any string
* a given string